### PR TITLE
Maintenance and new deployment method

### PR DESCRIPTION
**Pull request by pjanik**
[[#185353277]](https://www.pivotaltracker.com/story/show/185353277)

This PR is similar to recent work in Wildfires. It updates most of the dependencies and the deployment method. I focused on ThreeJS and React Three Fiber updates, and I skipped the MaterialUI upgrade, as this would be the most time consuming and likely to break things. I had to skip some Cypress tests, they work locally, but they never worked in any CI environment - I'll ask Tejal or Saravanan to take a look at it later.

### CODE REVIEW

File: src/components/header.tsx [hunk 1]
```diff
@@ -1,4 +1,4 @@
 import React from "react";
 import css from "./header.scss";
 
-export const Header: React.FC = ({ children }) => <div className={css.header}>{ children }</div>;
+export const Header: React.FC = ({ children }: { children?: JSX.Element | string }) => <div className={css.header}>{ children }</div>;
```
Comment (dougmartin) [inline src/components/header.tsx:4]: You can also use `React.ReactNode` as the children type and it handles all the variant cases.

File: src/components/view-3d/gauges.tsx [hunk 2]
```diff
@@ -39,5 +39,5 @@ export const Gauges: React.FC = observer(function WrappedComponent() {
         />;
       })
     }
-  </>;
+         </>;
```
Comment (dougmartin) [inline src/components/view-3d/gauges.tsx:42]: Is this to satisfy an eslint indentation check? It looks kind of weird.

**Review by dougmartin — APPROVED** [review]
Looks good :+1:

File: src/components/view-3d/gauges.tsx (hunk 2 above)
Comment (pjanik) [reply src/components/view-3d/gauges.tsx:42]: Hm, true, didn't notice it. It was actually done automatically by eslint --fix. Apparently, it want's the closing tag adjusted.

**Comment by pjanik** [comment]
Thanks, I've addressed your comments in the last commit.

**Pull request merged**